parser/src/conversion/block.rs

```diff
@@ -162,10 +162,11 @@ where
 }
 
 fn parse_scale(pair: &Pair<Rule>) -> Result<u8, Error> {
-    let input = if pair.as_str().ends_with('%') {
-        &pair.as_str()[..pair.as_str().len() - 1]
+    let input = pair.as_str().trim();
+    let input = if let Some(percentage) = input.strip_suffix('%') {
+        percentage
```

Comment:
seems like there can also be a space between number and percent sign

```suggestion
        percentage.trim_end()
```